.dockerignore

```diff
@@ -16,9 +16,6 @@ _kubevirtci
 kubevirtci
 
 # No need for source code is already compiled
-pkg
-cmd
-vendor
```

Comment:
Can we delete them from the container after the build is done?

Reply:
Sorry, brainfart, it is multistage and it never gets to the second stage of course.